### DISCUSSION TITLE
More Control Options for Chromecast Feature

### DISCUSSION THREAD

**Discussion by windmueller** [ente-io/ente#1699] [Enhancements]
The newly introduced Chromecast support is a great feature, bringing ente to my TV screen.

However, I would like to have more control over how the photos are displayed, Currently all photos will be shown randomly in a slideshow with a fixed interval.

For presenting a vacation album for example it would be great to be able to display the photos sorted by date. This could be a simple option in the share screen.

Another possibility would be to provide full control over the mobile device, e.g. swiping to the next photo, going back. etc.

**Comment by alnetloc** [comment]
Also, would be great to have the option to suppress captions for slideshows...especially if you are going to narrate the slide, the text just clutters the image.
